### ISSUE TITLE
Tagging in note text Script breaks the Table preview

### ISSUE THREAD

**Issue by ghost** [qownnotes/scripts#101]
I use the @tag tagging note text with #tag. When you use it like this it breaks the table preview. 
When you use @tag it works. I know i know # is not optimal for tagging but in 1writer they use for tagging the notes so thats my problem.

Edit: its enough when you set the # in the Tag word maker setting

**Comment by pbek** [comment]
Can you please mention the authors of the script (like advised in the issue template). You will find the authors in the *Script repository*.

**Comment by svenson26** [comment]
> Can you please mention the authors of the script (like advised in the issue template). You will find the authors in the _Script repository_.

I‘ve inform the author of the script

**Comment by Maboroshy** [comment]
Could you provide an example. 

I've set `#` as a tag word marker and put the following text into the note:
```
| 1 | 2 | 3 |
|---|---|---|
| #tag | 2 | 3 |
```
I got editor syntax highlighting, rendered note preview and the tag all working with QOwnNotes 20.4.12 on my Linux system.

**Comment by svenson26** [comment]
> Could you provide an example.
> 
> I've set `#` as a tag word marker and put the following text into the note:
> 
> ```
> | 1 | 2 | 3 |
> |---|---|---|
> | #tag | 2 | 3 |
> ```
> 
> I got editor syntax highlighting, rendered note preview and the tag all working with QOwnNotes 20.4.12 on my Linux system.

Not yet but tomorrow i can do. I work on Win10 with Qownnotes version 20.6.11

**Comment by svenson26** [comment]
It is enough to set a hashtag in the tag word marker and the table preview will not work anymore. As soon as you use for example an @ the preview will work again.
![Image 1](https://user-images.githubusercontent.com/67114383/86154110-25f6ba80-baf2-11ea-9488-6b2c6223d321.jpg)
![Image 2](https://user-images.githubusercontent.com/67114383/86154113-2727e780-baf2-11ea-904d-29e1b1452f23.jpg)

**Comment by Maboroshy** [comment]
I've updated QON and I can reproduce your issue. Maybe it's some kind of minor deprecation.
The only part of the script that affects html is the tag highlighting feature of the script. I've made the highlighting color setting  empty at that resolved the issue.
Try if it works for you.

**Comment by svenson26** [comment]
> 
> 
> I've updated QON and I can reproduce your issue. Maybe it's some kind of minor deprecation.
> The only part of the script that affects html is the tag highlighting feature of the script. I've made the highlighting color setting empty at that resolved the issue.
> Try if it works for you.

Yes it works when i empty the color setting. The preview shows the table correct

**Comment by Maboroshy** [comment]
I've updated the script. Please, test if it fixes the issue.
